### PR TITLE
Azure Uploads on Staging

### PR DESCRIPTION
**Pull request by goldblatt**
### Description
Reverts a prior change since isDev also includes isStaging
Adds a fix to the zone since the zone is erroring

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public FileUploadController(
   }
 
   public Result index(Request request) {
-    if (!(isDevEnvironment() || isStaging())) {
+    if (!isDevEnvironment()) {
```
Comment (bion) [inline universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java:53]: question (blocking): do we not want this on staging?

**Review by bion — CHANGES_REQUESTED** [review]

**Comment by goldblatt** [comment]
Note that we discussed binding the zone id at a higher level and injecting into this code. will do that in a follow up PR confirming that this will work on the azure set up

File: universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java (hunk 1 above)
Comment (goldblatt) [reply universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java:53]: ahh yeah so i looked back at the isDevEnvironmnet and it does the isStaging request

Comment (goldblatt) [reply universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java:53]: https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/controllers/dev/DevController.java#L29

Comment (bion) [reply universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java:53]: right on

**Review by bion — APPROVED** [review]

**Review by zaralouis-sf — APPROVED** [review]

**Pull request merged**